backend/src/main/scala/akka/viz/events/EventPublisherActor.scala

```diff
@@ -10,8 +10,9 @@ class EventPublisherActor extends Actor with ActorLogging {
   var subscribers = immutable.Set[ActorRef]()
   var availableTypes = immutable.Set[Class[_ <: Any]]()
   var eventCounter = 0L
+  var replayables: Vector[ReplayableEvent] = Vector.empty
```

Comment:
I see two problems here, first is that that collection is unbound, second is that you keep Instantiated and reference to Actor which can't be GCed.

Probably those events could be packed into single events, keep only the last one for the ActorRef (it could be instantiated and killed many times?) And send them in one batch. Using weak reference might be also a good idea, to play nice with GC.